lib/quickbooks/service/item.rb

```diff
@@ -8,18 +8,16 @@ def delete(item)
       end
 
       def url_for_resource(resource)
-        url = super(resource)
-        "#{url}?minorversion=#{Quickbooks::Model::Item::MINORVERSION}"
+        super(resource)
       end
 
       def fetch_by_id(id, params = {})
-        url = "#{url_for_base}/item/#{id}?minorversion=#{Quickbooks::Model::Item::MINORVERSION}"
+        url = "#{url_for_base}/item/#{id}"
         fetch_object(model, url, params)
       end
 
       def url_for_query(query = nil, start_position = 1, max_results = 20, options = {})
-        url = super(query, start_position, max_results, options)
-        "#{url}&minorversion=#{Quickbooks::Model::Item::MINORVERSION}"
+        super(query, start_position, max_results, options)
```

Comment:
Seems like this and `url_for_resource` could be dropped in favor of the parent class's implementation now.